uvtools/dspec.py

```diff
@@ -1250,12 +1309,12 @@ def sinc_downweight_mat_inv(nchan, df, filter_centers, filter_widths,
         channel width (Hz)
     filter_centers: float or list
         float or list of floats of centers of delay filter windows in nanosec
-    filter_widths: float or list
+    filter_half_widths: float or list
         float or list of floats of widths of delay filter windows in nanosec
```

Comment:
Should the docstring be updated to reflect the fact that it's a half-width? Also what do you mean by half-width rather than full-width?

Reply:
filter_widths always meant half_widths. I've changed the variable name to accurately reflect this.

Reply:
You are correct to point out that the docstring never correctly specified this. I've changed it.